services/minio/internal/cmd/meta.go

```diff
@@ -26,6 +27,16 @@ var Metadata = def.Metadata{
 			Get:    true,
 			List:   true,
 		},
+
+		Create: []def.Pair{
+			def.PairLocation,
+		},
+		Delete: []def.Pair{
+			def.PairLocation,
```

Comment:
Maybe `Location` is not a necessary parameter for `RemoveBucket`?

Reply:
No, not always necessary, but it's optional. Should optional parameter be here, or somewher else?

Reply:
The optional parameters are indeed specified here. There are no optional parameters for `RemoveBucket` at this time from the [doc](https://docs.min.io/docs/golang-client-api-reference.html#RemoveBucket)?